src/helpers/parser/listItemParser.js

```diff
@@ -173,15 +173,17 @@ const parseTours = (data, skipLastDivider) => {
   }));
 };
 
-const parseCategories = (data, skipLastDivider) => {
+const parseCategories = (data, skipLastDivider, routeName = 'Category') => {
```

Comment:
instead of hardcoding the route string here, `ScreenName.Category` could be used.

Reply:
change has been made